### PR TITLE
Add volumes to make running buildah within a container easier

### PR DESCRIPTION
**Pull request by rhatdan**
[NO TESTS NEEDED] Since this is just changing Dockerfiles for use on
quay.io.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

#### What type of PR is this?

> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test 
> /kind feature
> /kind flake
> /kind other

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
You missed one:

`https://github.com/containers/buildah/blob/master/contrib/buildahimage/stablebyhand/Containerfile.buildahstable`

**Comment by rhatdan** [comment]
@containers/podman-maintainers PTAL
@vrothberg @giuseppe @saschagrunert PTAL

File: contrib/buildahimage/stable/Dockerfile [hunk 1]
```diff
@@ -23,6 +23,9 @@ RUN mkdir -p /var/lib/shared/overlay-images /var/lib/shared/overlay-layers /var/
 RUN echo build:2000:50000 > /etc/subuid \
  && echo build:2000:50000 > /etc/subgid
 
+VOLUME /var/lib/containers
+VOLUME /home/buildah/.local/share/containers
```
Comment (saschagrunert) [inline contrib/buildahimage/stable/Dockerfile:27]: Should we use `build` here?
```suggestion
VOLUME /home/build/.local/share/containers
```

Comment (rhatdan) [reply contrib/buildahimage/stable/Dockerfile:27]: Yikes good catch @saschagrunert

**Review by saschagrunert — APPROVED** [review]
LGTM

**Comment by TomSweeneyRedHat** [comment]
LGTM
Got a test flake, I've restarted.  Probably could have tagged this with [CI:DOCS].

**Pull request merged**